### DISCUSSION TITLE
mermaid.mermaidOptions is not allowed (v2.2.0)

### DISCUSSION THREAD

**Discussion by dprothero** [facebook/docusaurus#8268] [Q&A] [answered]
According to [the documentation](https://docusaurus.io/docs/markdown-features/diagrams#configuration), I should be able to specify options to pass to mermaid.js:

```javascript
module.exports = {
  themeConfig: {
    mermaid: {
      mermaidOptions: {
        maxTextSize: 50,
      },
    },
  },
};
```

However, when I run `yarn start` I'm getting: `"mermaid.mermaidOptions" is not allowed`

[full-output.txt](https://github.com/facebook/docusaurus/files/9902899/full-output.txt)

If I comment out the `mermaidOptions` key, everything works and mermaid diagrams render fine (with the default options, of course).

When I search the repo, the only instance of `mermaidOptions` I find is in the documentation, so was this feature never implemented?

**Comment by slorber** [comment, marked as answer]
Doc fixed in https://github.com/facebook/docusaurus/pull/8272, you need to use "options" instead of "mermaidOptions"
